devtools/allopen/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.3.61'
+    id "org.jetbrains.kotlin.kapt" version "1.3.61"
+    id 'com.gradle.plugin-publish' version "0.10.1"
+    id 'java-gradle-plugin'
+}
+
+group 'io.quarkus'
+version '999-SNAPSHOT'
```

Comment:
I don't know exactly where the name of the plugin is defined but we should be careful about it.

I think having `kotlin-` somewhere would be necessary. Same for the Maven artifact id and the directory of the module.